java/ql/lib/semmle/code/java/frameworks/android/Intent.qll

```diff
@@ -52,3 +53,114 @@ class BundleGetterMethod extends Method, TaintPreservingCallable {
+        "android.os;BaseBundle;true;putStringArray;;;Argument[1];MapValue of Argument[-1];value",
+        "android.os;Bundle;true;getBinder;(String);;MapValue of Argument[-1];ReturnValue;value",
+        "android.os;Bundle;true;getBundle;(String);;MapValue of Argument[-1];ReturnValue;value",
+        "android.os;Bundle;true;getByteArray;(String);;MapValue of Argument[-1];ReturnValue;value",
+        "android.os;Bundle;true;getCharArray;(String);;MapValue of Argument[-1];ReturnValue;value",
+        "android.os;Bundle;true;getCharSequence;(String);;MapValue of Argument[-1];ReturnValue;value",
+        "android.os;Bundle;true;getCharSequence;(String,CharSequence);;Argument[1];ReturnValue;value",
+        "android.os;Bundle;true;getCharSequenceArray;(String);;MapValue of Argument[-1];ReturnValue;value",
+        "android.os;Bundle;true;getCharSequenceArrayList;(String);;MapValue of Argument[-1];ReturnValue;value",
+        "android.os;Bundle;true;getParcelable;(String);;MapValue of Argument[-1];ReturnValue;value",
+        "android.os;Bundle;true;getParcelableArray;(String);;MapValue of Argument[-1];ReturnValue;value",
+        "android.os;Bundle;true;getParcelableArrayList;(String);;MapValue of Argument[-1];ReturnValue;value",
+        "android.os;Bundle;true;getSerializable;(String);;MapValue of Argument[-1];ReturnValue;value",
+        "android.os;Bundle;true;getSparseParcelableArray;(String);;MapValue of Argument[-1];ReturnValue;value",
+        "android.os;Bundle;true;getStringArrayList;(String);;MapValue of Argument[-1];ReturnValue;value",
+        "android.os;Bundle;true;putAll;(Bundle);;MapKey of Argument[0];MapKey of Argument[-1];value",
+        "android.os;Bundle;true;putAll;(Bundle);;MapValue of Argument[0];MapValue of Argument[-1];value",
+        "android.os;Bundle;true;putBinder;;;Argument[0];MapKey of Argument[-1];value",
+        "android.os;Bundle;true;putBinder;;;Argument[1];MapValue of Argument[-1];value",
+        "android.os;Bundle;true;putBundle;;;Argument[0];MapKey of Argument[-1];value",
+        "android.os;Bundle;true;putBundle;;;Argument[1];MapValue of Argument[-1];value",
+        "android.os;Bundle;true;putByte;;;Argument[0];MapKey of Argument[-1];value",
+        "android.os;Bundle;true;putByteArray;;;Argument[0];MapKey of Argument[-1];value",
+        "android.os;Bundle;true;putByteArray;;;Argument[1];MapValue of Argument[-1];value",
+        "android.os;Bundle;true;putChar;;;Argument[0];MapKey of Argument[-1];value",
+        "android.os;Bundle;true;putCharArray;;;Argument[0];MapKey of Argument[-1];value",
+        "android.os;Bundle;true;putCharArray;;;Argument[1];MapValue of Argument[-1];value",
+        "android.os;Bundle;true;putCharSequence;;;Argument[1];MapValue of Argument[-1];value",
+        "android.os;Bundle;true;putCharSequenceArray;;;Argument[1];MapValue of Argument[-1];value",
+        "android.os;Bundle;true;putCharSequenceArrayList;;;Argument[1];MapValue of Argument[-1];value",
```

Comment:
Is the model for `Argument[0]` -> `MapKey of Argument[-1]` missing in these three rows?